cmd/src/server.rs

```diff
@@ -448,7 +452,7 @@ impl<ER: RaftEngine> TiKVServer<ER> {
         );
 
         // Create cdc.
-        let mut cdc_worker = Box::new(tikv_util::worker::Worker::new("cdc"));
+        let mut cdc_worker = self.background_worker.lazy_build("cdc");
```

Comment:
I think cdc worker should be run in an individual thread, sometime it would even be a bottleneck after we enable old  value (so we had a plan to scale it to multiple threads)

Reply:
But it is a pool rather than a thread.... You can increate the size of the pool.

Reply:
I don't think putting a worker with a heavy load with other workers is a good idea, it may block others.

Reply:
Emmmm. Ok, I will let cdc-endpoint be a independent pool.

Reply:
But, is there some configuration to enable or disable cdc? Do we have to create the cdc-endpoint even if user does not need cdc?

Reply:
We don't have such a config currently..... It's reasonable, will file an issue for this.

Reply:
FYI, https://github.com/tikv/tikv/issues/8959